### DISCUSSION TITLE
Nemo - shortcut for copying file name

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#135] [Ideas]
Currently, file names can be copied quickly by pressing F2 and then CTRL+C, however this only works on non-write-protected mountpoints.

On files that are on write-protected mountpoints, the fastest way is to open the "Properties" window and manually select the file name from there. Please add a shortcut to copy the file name.

**Comment by Secret-chest** [comment]
File path too.

**Reply by HT-7** [reply]
It already is copied when pressing CTRL+C or or clicking the top menu > Edit > Copy.
